js/binance.js

```diff
@@ -3722,6 +3722,7 @@ module.exports = class binance extends Exchange {
         };
         let method = undefined;
         if (market['linear']) {
+            // ? The premium index and funding rate are different things, does this method work correctly?
```

Comment:
this probably requires standalone testing and maybe a standalone fix/pr

Reply:
I removed this comment